### PR TITLE
Bumping to HTML5 & adding device friendly meta tag

### PR DESCRIPTION
**Pull request by mrphs**
Because why not :)

### CODE REVIEW

**Comment by mrphs** [comment]
But in all seriousness, this small change makes the viewing of the service page on smaller monitors and on modern devices easier.

File: html.go [hunk 1]
```diff
@@ -6,12 +6,12 @@ import (
...
 // IndexPage contains the landing page of the service.  It implements a simple
 // form that asks for an IP address and TCP port.
-var IndexPage = `<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN"
-    "http://www.w3.org/TR/html4/strict.dtd">
+var IndexPage = `<!DOCTYPE html>
 <html lang="en">
 
 <head>
-  <meta http-equiv="content-type" content="text/html; charset=utf-8">
+  <meta charset=utf-8">
```
Comment (NullHypothesis) [inline html.go]: There's a missing " before utf-8.

**Review by NullHypothesis — CHANGES_REQUESTED** [review]
Thanks for the patch, Nima! I only have one minor suggestion. Other than that, it looks good.

**Comment by mrphs** [comment]
Ah you're right. The new github confuses me so I fixed it with a squash and forced push :)

**Comment by NullHypothesis** [comment]
Thanks! Merged and deployed.

**Pull request merged**